### PR TITLE
Eng 1910

### PR DESCRIPTION
**Pull request by gesparza3**
This PR fixes the tests under `local` to work with previously configured modules. Most of the fixes revolved around fixing updated module inputs. I also added a test for the recently created `prometheus-operator` module.

### CODE REVIEW

File: tests/local/local_test.go [hunk 1]
```diff
@@ -102,34 +94,22 @@ func TestSetup(t *testing.T) {
...
-	t.Run("MetricsServer", common.MetricsServerTest)
+
+	//t.Run("SDM", testLeadSdm)
+	//t.Run("KubeResourceReport", common.KubeResourceReportTest)
+	//t.Run("ExternalDNS", common.ExternalDnsTest)
+	//t.Run("KubeDownscaler", common.KubeDownscalerTest)
+	//t.Run("K8sSpotTerminationHandler", common.K8sSpotTerminationHandlerTest)
+	//t.Run("KubeJanitor", common.KubeJanitorTest)
+	//t.Run("MetricsServer", common.MetricsServerTest)
+	//t.Run("PrometheusOperator", common.TestPrometheusOperator)
```
Comment (mrparkers) [inline tests/local/local_test.go:105]: did you intend to leave this commented out?

Comment (gesparza3) [reply tests/local/local_test.go]: Definitely did not, I think my formatting fix I pushed up inadvertently included this version from testing the `lead-dashboard` module. Just pushed up a better copy, sorry about that.

File: tests/common/prometheus-operator.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"testing"
+)
+
+func TestPrometheusOperator(t *testing.T) {
+	t.Parallel()
+	kubeconfig := TestModuleGetStringGlobal(t, KubeConfigPath)
+
+	// LEAD HARBOR
+	testDashboard := TestModule{
```
Comment (ChrisSchreiber) [inline tests/common/prometheus-operator.go:12]: Should update command and test module instance name to reflect test

Comment (gesparza3) [reply tests/common/prometheus-operator.go]: Done

**Pull request merged**